src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;

```

Comment:
Better to use `double` type for `radius` and side lengths in other figures for consistency.